style/dashboards.css

```diff
@@ -17,9 +17,13 @@
   background: #326AA7;
   color: white;
 }
+.dashboard-list .list-group-item {
+  border: none;
+  border-top: 1px solid #ddd;
+}
 .big-dashboard-list .list-group-item .dashboard-description { font-size: 18px; }
 .scrollable-dashboard-list {
```

Comment:
You've removed the only two `.scrollable-dashboard-list` in the repo, so we can safely remove their styles.